server/app/rest_api/rest.py

```diff
@@ -147,13 +192,16 @@ class AnnotationsObsAPI(Resource):
         }
     )
     def get(self):
+        content_type = JSON_MIMETYPE
         fields = request.args.getlist("annotation-name", None)
         try:
             annotation_response = current_app.data.annotation({}, "obs", fields)
         except KeyError:
             return make_response(f"Error bad key in {fields}", HTTPStatus.BAD_REQUEST)
         try:
-            return make_response(jsonify(annotation_response), HTTPStatus.OK)
+            return make_response(
+                annotation_response, HTTPStatus.OK, {"Content-Type": content_type}
```

Comment:
I don't find this style (use of `content_type`) to be helpful.   If we are going to move away from strings, it should be to names that signify intent (eg, `HTTPStatus.BAD_REQUEST`).

So I would either leave this as a string ("application/json"), or I would use `JSON_MIMETYPE` directly.

Reply:
I was anticipating your additions where content_type would be set to BINARY_MIMETYPE earlier. Happy to make your changes if you had a different way of handling this.

Reply:
Ah, I see.  I thought it would be clearer to have two separate branches which each called make_response.  That would be more stylistically aligned with the rest of the code.  

In other words, something link:
```
   if accept_type == "application/octet-stream":
       make_ressponse(current_app.data.doitReturningFBS({}, "obs", fields), HTTPStatus.OK, {"Content-Type": "application/octet-stream"})
   elif accept_type == "application/json":
       make_ressponse(current_app.data.doItReturningJSON({}, "obs", fields), HTTPStatus.OK, {"Content-Type": "application/json"})
   else:
       make_response("Unsupported MIME types", HTTPStatus.NOT_ACCEPTABLE)
```

that said, I'm not feeling dogmatic!

Reply:
I made the change, I was mis-predicting what you would be doing when combing the routes for the flatbuffer PR.